src/Extensions/Abstractions/IMachineInformation.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Net;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Machine information interface</summary>
+	public interface IMachineInformation
+	{
+		/// <summary>Agent name</summary>
+		string AgentName { get; }
+
+		
+		/// <summary>Build version string</summary>
+		string BuildVersion { get; }
+
+
+		/// <summary>The deployment slice</summary>
+		string DeploymentSlice { get; }
+
+
+		/// <summary>The environment name</summary>
+		string EnvironmentName { get; }
+
+
+		/// <summary>Is Canary environment</summary>
+		bool IsCanary { get; }
+
+
+		/// <summary>Is Dev Fabric environment</summary>
+		bool IsDevFabric { get; }
```

Comment:
I suggest removing this property entirely - or at the very least renaming it. `Fabric` name is legacy. Also .net provides the dev environment flag now.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.hostingenvironmentextensions.isdevelopment?view=aspnetcore-3.1

Reply:
Removed, thanks